data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/disable/disable_ability.cfg

```diff
@@ -0,0 +1,30 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [disable]
+##
+# Actions:
+# Give the leaders a disables ability
+##
+# Expected end state:
+# Nothing is disabled
```

Comment:
expected end state is that the unit with ability has all attacks disabled

active_on: one of defense or offense; if left out, the special is active on both.
apply_to: one of self,opponent,attacker,defender,both (default: self). Determines who the effects of this special are applied to.

Reply:
That's what I would expect, but that's not the current state - the current state is that putting [disable] as an ability does not disable anything.

Reply:
@newfrenchy83 can you confirm whether this is intentional?

Reply:
i pull a PR for fix issue showed by the test.

Reply:
> That's what I would expect, but that's not the current state - the current state is that putting [disable] as an ability does not disable anything.

I think in case of bug there was discussion to add markup or comment that it is not the actual expected result.

Reply:
@newfrenchy83 it still doesn't appear to work - it shows on the UI, but doesn't actually prevent using the attacks:
![Screenshot from 2024-08-11 23-10-52](https://github.com/user-attachments/assets/8c63ed96-526d-4faa-bf52-adeb696d2c5c)

Reply:
perhaps what you should give disable abiliity to one fighter only with apply_to=opponent, else are you sure what [disable] special was not disabled of harcode?